src/components/components/ebay-notice-base/component.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    onCreate() {
+        this.state = {
+            dismissed: false,
+        };
+    },
+    handleDismissClick: forwardEvent('on-dismiss'),
+    handleDismissKeydown: forwardEvent('on-dismiss'),
```

Comment:
This is the problem. It should just be `dismiss` and not `on-dismiss`. The event it emits should be without the `on-`

Reply:
Gotcha. The `on-` is implied. Good to know. I fixed this.